pkg/api/types.go

```diff
@@ -1239,6 +1239,11 @@ func (m *MasterProfile) IsVHDDistro() bool {
 	return m.Distro == AKSUbuntu1604 || m.Distro == AKSUbuntu1804
 }
 
+// IsAvailabilitySets returns true if the customer specified disks
+func (m *MasterProfile) IsAvailabilitySets() bool {
```

Comment:
this is trivial but can you add a small unit test to make sure we maintain full coverage?

Reply:
Of course, and I can probably add one to cover the "master VMAS / nodepool VMSS scaling" case.